host/xtest/CMakeLists.txt

```diff
@@ -16,15 +16,20 @@ if(OPENSSL_FOUND)
 	set (OPENSSL_PRIVATE_LINK OpenSSL::Crypto)
 endif()
 
+find_package(Python COMPONENTS Interpreter Development)
+if(NOT Python_Interpreter_FOUND)
+	message(FATAL_ERROR "Python interpreter has not been found")
+endif()
+
 include(GNUInstallDirs)
 
 macro(EMBED_8100FILE prefix infile)
 	add_custom_command(
 		OUTPUT  regression_8100_${prefix}.h
-		COMMAND ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py --inf ${infile}
+		COMMAND ${Python_EXECUTABLE} ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py --inf ${infile}
 	--out ${CMAKE_CURRENT_BINARY_DIR}/regression_8100_${prefix}.h
 	--name regression_8100_${prefix}
-		DEPENDS ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py ${infile}
+		DEPENDS ${Python_EXECUTABLE} ${OPTEE_TEST_ROOT_DIR}/scripts/file_to_c.py ${infile}
```

Comment:
I doubt this dependency is needed since we have the `Python_Interpreter_FOUND` test above.

Reply:
looks to be redundant as we are again depending on the same .py file
could be removed from my perspective